xamarin/Xamarin.Forms
# Fix to display toolbar icons under UWP Shell (partial fix for #11706).

### Description of Change ###

The ShellToolbarItemTemplate for UWP was using incorrect binding for the IconImageSource, resulting in 
`Error: Converter failed to convert value of type 'Xamarin.Forms.Internals.AsyncValue`1<Windows.UI.Xaml.Media.ImageSource>' to type 'ImageSource'; BindingExpression: Path='IconImageSource' DataItem='Xamarin.Forms.ToolbarItem'; target element is 'Windows.UI.Xaml.Controls.Image' (Name='null'); target property is 'Source' (type 'ImageSource').
`
Also, after correcting the binding error, the icon was displayed above the text and was very small, so I moved it to the left of the text.

### Issues Resolved ### 

- partially fixes #11706

### API Changes ###
 None

### Platforms Affected ### 
- UWP

### Behavioral/Visual Changes ###
Toolbar items under UWP Shell can now display both icon and text.

### Before/After Screenshots ### 
<img width="316" alt="11706-before" src="https://user-images.githubusercontent.com/1275686/90599110-22ee8100-e1a9-11ea-9aa0-9f9d664169e0.png">
<img width="316" alt="11706-after" src="https://user-images.githubusercontent.com/1275686/90599113-23871780-e1a9-11ea-8492-368628e2399d.png">

### Testing Procedure ###
1. Create a default Xamarin.Forms app using Shell.
2. Add UWP project
3. Add toolbar items to the main page:
`<ContentPage.ToolbarItems> <ToolbarItem Text="Add" Clicked="AddItem_Clicked" IconImageSource="tab_feed.png"/> <ToolbarItem Text="Secondary" Order="Secondary" IconImageSource="tab_about.png"/> </ContentPage.ToolbarItems>
`
When the UWP app is started, the toolbar should display both image and text.

### PR Checklist ###
